### PR TITLE
Orv2 1678 Add static home page under path /home

### PR DESCRIPTION
**Pull request by chrisrobbo-aot**
#Summary
Added folder static_home_page which gets added to the default frontend container

image

# Description
This folder gets copied to the /usr/share/nginx/html folder
The nginx.conf is modified to direct the path /home to the Home.html static page in this folder

All assets used by this page are local to static_home_page

Dependencies are only the links to which the page links (eg. onRoute login, corporate web pages etc.)

- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Deployed the application. Ran <base-url>/home . Checked that the home page was displayed. Clicked on links, checked that they resolved OK in new tabs

## Checklist

- [x ] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ x] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [x ] I have added tests that prove my fix is effective or that my feature works
- [x ] New and existing unit tests pass locally with my changes
- [x ] Any dependent changes have already been accepted and merged

## Further comments

#todo the links to  the onRoute logins should be variables for each of the environments dev /test/ /demo etc. 

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://onroutebc-1242-frontend.apps.silver.devops.gov.bc.ca)
- [Vehicles](https://onroutebc-1242-vehicles.apps.silver.devops.gov.bc.ca/api)
- [Dops](https://onroutebc-1242-dops.apps.silver.devops.gov.bc.ca/api)
- [TPS-Migration](https://onroutebc-1242-tps-migration.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/onroutebc/actions/workflows/analysis.yml)

After merge, new images are promoted to:
- [Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge.yml)

### CODE REVIEW

**Review by erikataot — APPROVED** [review]

File: frontend/static_home_page/home.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+            <p>The Ministry of Transportation and Infrastructure's commercial vehicle permitting system is transitioning 
+              to a new system called onRouteBC. The first phase of onRouteBC includes support for Term Oversize permits, 
+              payable by credit card only. Over time onRouteBC will offer more permit types and payment methods. During 
+              transition, permits that are not available in onRouteBC continue to be available from the existing system 
+              by contacting the Provincial Permit Center.</p>
+            <div class="homeContentBoxContent2Col">
+              <div class="homeBorderedContentCol">
+                  <h4>Term Oversize Permits Only</h4>
+                  <p>Login with your Business BCeID and pay online with your credit card</p>
+                  <a role="button" class="btn btnBlue" href="https://onroutebc-prod-frontend.apps.silver.devops.gov.bc.ca" target="_blank">
```
Comment (krishnan-aot) [inline frontend/static_home_page/home.html:44]: Does it need to be parameterized with an env variable so that it can point to the right domain?

Comment (zgong-gov) [reply frontend/static_home_page/home.html:44]: Good point, but is there an effective way to do that in static html without the use of JavaScript?

Comment (erikataot) [reply frontend/static_home_page/home.html:44]: yeah - not a bad idea. It'll be a great ticket for a followup PR.

Comment (erikataot) [reply frontend/static_home_page/home.html:44]: Put a ticket in our backlog for this

Comment (chrisrobbo-aot) [reply frontend/static_home_page/home.html:44]: yes it should

**Pull request merged**